large_media

```diff
@@ -1 +1 @@
-Subproject commit 52bb06c5f75001d52fc7746a9df0274567079637
+Subproject commit eb7cc99c84687a89ee99287d8a9db7cae14ef28c
```

Comment:
**Caution!** This contains a submodule update

Reply:
How do I delete this changes without rolling back to a previous commit?